lib/rubocop/cop/rails/http_positional_arguments.rb

```diff
@@ -0,0 +1,89 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop is used to identify usages of `http methods
+      # like get,post,put,path` without the usage of keyword arguments
+      # in your tests and change them to use keyword arguments.
+      #
+      # @example
+      #   # bad
+      #   get :new, { user_id: 1}
+      #
+      #   # good
+      #   get :new, params: { user_id: 1 }
+      class HttpPositionalArguments < Cop
+        MSG = 'Use `keyword arguments` instead of ' \
```

Comment:
I was just following others.   Can you add what exactly you want from a MSG standpoint in a contributing doc to save future developers and yourself time?